pontoon/sync/formats/xliff.py

```diff
@@ -13,52 +14,30 @@ class XLIFFEntity(VCSTranslation):
     """
     Interface for modifying a single entity in an xliff file.
     """
-
-    def __init__(self, order, unit):
-        self.order = order
+    def __init__(
+        self,
+        key,
+        source_string,
+        source_string_plural,
+        strings,
+        comments=None,
+        order=None,
+        unit=None
```

Comment:
We don't seem to be using `unit` in `XLIFFEntity`, so I'd remove it.

OTOH, we should add `context` parameter, since we also had it before.